src/amo/pages/UserProfileEdit/index.js

```diff
@@ -494,7 +494,7 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                 isEditingCurrentUser
                   ? i18n.gettext('Account')
                   : i18n.sprintf(i18n.gettext('Account for %(username)s'), {
-                      username,
+                      username: user ? user.username : userId,
```

Comment:
So if we don't have a `user` we're going to display the `userId`? I guess that's all the information we have, but it doesn't seem particularly user friendly as the id will just be a meaningless number. Would it be better to just not display anything rather than this meaningless number?

Reply:
That's only for admins, I don't think it really matters.

Reply:
We could probably not display anything though.

Reply:
I changed to use `Account` when there is no user.